### PR TITLE
[5.4] Add make:view Artisan command

### PR DESCRIPTION
**Pull request by LKaemmerling**
[Orginal #14965 ]

Adds the ability to create Views from Artisan.

`php artisan make:view {NameOfView}` 

**Create Folder(s) and View:**

`php artisan make:view folder.to.thisView` creates an thisView.blade.php in the  _`view folder`_`/folder/to/` and create the folder _`view folder`_`/folder/to/` if it doesn't exists.

**Options:**
Of course, combinations of options are possible.

`php artisan make:view TestView` creates an TestView.blade.php in the _`view folder`_
Content:

```
@extends('layouts.app')

@section('content')
<div class='container'>

</div>
@endsection
```

`php artisan make:view TestView --parent=OtherParentLayout` creates an TestView.blade.php in the _`view folder`_
Content:

```
@extends('OtherParentLayout')

@section('content')
<div class='container'>

</div>
@endsection
```

`php artisan make:view TestView --section=OtherSection` creates an TestView.blade.php in the _`view folder`_

```
@extends('layouts.app')

@section('OtherSection')
<div class='container'>

</div>
@endsection
```

`php artisan make:view TestView --class=OtherClass` creates an TestView.blade.php in the _`view folder`_

```
@extends('layouts.app')

@section('content')
<div class='OtherClass'>

</div>
@endsection
```

`php artisan make:view TestView --class=false` creates an TestView.blade.php in the _`view folder`_

```
@extends('layouts.app')

@section('content')

@endsection
```

### CODE REVIEW

File: src/Illuminate/Foundation/Console/stubs/view.stub [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@extends('DummyParentView')
+
+@section('DummySection')
+<DumyDiv></DumyDiv>
+@endsection
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/view.stub]: missing newline

**Comment by fernandobandeira** [comment]
Related issue #14517

**Comment by JackaThompson** [comment]
I've actually just created a package for this: I'm working on updates as we speak for new features. It doesn't include tests yet: https://github.com/jackathompson/bladeviews

**Comment by taylorotwell** [comment]
Holding off on including this in the core.

**Comment by LKDevelopment** [comment]
@taylorotwell 
Why? What is the justification for everything else to have a generator but not for the views?

**Pull request closed without merging**